core/src/main/java/org/pentaho/cassandra/util/CassandraUtils.java

```diff
@@ -77,7 +77,7 @@
   }
 
   public static class CQLOptions {
-    public static final String CQLVERSION_OPTION = "cqlVersion"; //$NON-NLS-1$
+    public static final String CQLVERSION_OPTION = "driverVersion"; //$NON-NLS-1$
```

Comment:
Would it make sense to change the constant name as well? Not sure if it's safe...